quickwit-indexing/src/actors/indexer.rs

```diff
@@ -404,7 +404,7 @@ mod tests {
     use std::time::Duration;
 
     use quickwit_actors::{create_test_mailbox, Universe};
```

Comment:
in the indexer actor, we can keep the sort attribute in the indexer state but always set it to None.